### PR TITLE
Make cucumber tests pass and run them in CI

### LINKED ISSUE #18: Cucumber tests might not be triggered
There appear to be cucumber tests in the source code, [here](https://github.com/Australian-Genomics/CTRL/tree/e8967f79874459500b2887851cc96275d79af699/features/consent). But I'm not sure the rspec tests are running them. There is no documentation on how to run these either.

A satisfactory resolution to this issue would be to document how to run the cucumber tests in the readme and run them in CI, if they aren't already being run.

### PR DESCRIPTION
**Pull request by DoxasticFox**
Resolves #18.

Important things to note in this PR:

* Cucumber tests are running in CI.
* I deleted lots of tests which are no longer needed.
* I disabled tests (search for `TODO(DoxasticFox)` in this PR) which I couldn't get to run easily. I raised a follow-up ticket (Australian-Genomics/CTRL#30) to re-enable one of those tests. Re-enabling the other test should be covered by Australian-Genomics/CTRL#5.
* There's some random bug-fixes to the tests and the app, but they're not that important. Just click approve :wink:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -299,12 +299,21 @@ You can create the initial admin user by visiting `localhost:3000/subfolder/refi
 
 ## <a id="testing"></a> Testing
 
-We use [Capybara](https://github.com/teamcapybara/capybara) and [Rspec](https://rspec.info/) for our unit tests. Type and enter `rspec` in your terminal console to run the tests. Or, if using docker:
+We use [Capybara](https://github.com/teamcapybara/capybara) and [Rspec](https://rspec.info/) for our unit tests.
+
+To run the `cucumber` tests, enter `cucumber` in your terminal console. Or, if using docker:
+
+```shell
+docker-compose --env-file=.env.dev bundle exec rspec
```
Comment (DoxasticFox) [inline README.md:307]: From @mathob: 
```suggestion
docker-compose --env-file=.env.dev run web bundle exec rspec
```

File: README.md [hunk 2]
```diff
@@ -299,12 +299,21 @@ You can create the initial admin user by visiting `localhost:3000/subfolder/refi
 
 ## <a id="testing"></a> Testing
 
-We use [Capybara](https://github.com/teamcapybara/capybara) and [Rspec](https://rspec.info/) for our unit tests. Type and enter `rspec` in your terminal console to run the tests. Or, if using docker:
+We use [Capybara](https://github.com/teamcapybara/capybara) and [Rspec](https://rspec.info/) for our unit tests.
+
+To run the `cucumber` tests, enter `cucumber` in your terminal console. Or, if using docker:
+
+```shell
+docker-compose --env-file=.env.dev run web bundle exec rspec
```
Comment (DoxasticFox) [inline README.md:307]:
```suggestion
docker-compose --env-file=.env.dev run web bundle exec rake cucumber
```

**Review by mathob — APPROVED** [review]
On a laptop CTRL docker installation: by following steps in .github/workflows/pipeline.yml I was able to run cucumber tests successfully

**Comment by DoxasticFox** [comment]
@mathob Thanks for the review! Now it looks like it's failing because I broke `origin/main` in #33. #41 should fix it, and it's only has 3 new lines. Could I get a quick review on that before merging this? 🙏

**Comment by DoxasticFox** [comment]
@mathob Actually I also need #38, but @thangu11 is taking a look

**Pull request merged**